config/dependency-lint.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+module.exports = {
+  generateTests: false,
+  allowedVersions: {
+    // https://github.com/jasonmit/ember-cli-moment-shim/issues/183
+    moment: "2.24.0",
+
+    // EC v3+ heavily relies on Proxy which is not supported by IE11
+    "ember-changeset": "^2.2.4",
+    "ember-changeset-validations": "^2.2.1",
```

Comment:
Should we also list the GraphQL dependency mentioned in the dependabot config here?

Reply:
Not sure. Those dependencies in here should not be updated. However, `graphql-tools` should be updated but I didn't have the time to do it right now..